Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -179,15 +180,17 @@ namespace internal {
     typedef Patch_id&                           reference;
     typedef boost::read_write_property_map_tag  category;
 
-    Connected_components_pmap()
-      : patch_ids_map()
-    {}
+    Connected_components_pmap(PM& pmesh)
+    {
+      patch_ids_map = add(boost::face_property_t<Patch_id>("PMP_patch_id"), pmesh);
+      remove(patch_ids_map, pmesh);
+    }
```

Comment:
I am not very happy with this hack (add and remove) that I found. It is needed because of a compilation error due to the fact that `boost::property_map<...>` does not have a default constructor.
If one of you has an idea, I'll take it!
@afabri @sloriot @lrineau ?